.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+version: 2.1
+jobs:
+  test:
+    parameters:
+      gemfile:
+        type: string
+      ruby_version:
+        type: string
+    docker:
+      - image: salsify/ruby_ci:<< parameters.ruby_version >>
+      - image: circleci/postgres:9.6
```

Comment:
I wonder if we should move to pg 12 since the final release of 9.6 is this year.

Reply:
Yes! One thing at a time though :)